Error when running the tests in an Angular Wrappers project

### Description  
When you create an Angular Wrappers project and run `ig test` the following error is thrown:
ERROR in ./node_modules/rxjs/Rx.js
Module not found: Error: Can't resolve 'rxjs-compat'

- Ignite UI CLI version: 4.3.x
- Framework: Angular
- Project type: ig-ts

### Steps to reproduce  

1. Create a new ig-ts project
2. Add a component
3. Run `ig test`
4. Take a look at the console

### Result  
ERROR in ./node_modules/rxjs/Rx.js
Module not found: Error: Can't resolve 'rxjs-compat' in 'C:\New folder\newIgniteUIAngular\node_modules\rxjs'
 @ ./node_modules/rxjs/Rx.js 6:9-31
 @ ./karma-test-entry.ts

### Expected result  
No error


### Attachments  
Attach a sample if available, and screenshots, if applicable.

--- a/templates/angular/ig-ts/projects/empty/files/karma-test-entry.ts
+++ b/templates/angular/ig-ts/projects/empty/files/karma-test-entry.ts
@@ -1,5 +1,5 @@
 import 'core-js';
-import 'rxjs/Rx';
+import 'rxjs';
 import 'zone.js/dist/zone';
 import 'zone.js/dist/long-stack-trace-zone';
 import 'zone.js/dist/async-test';

